query.go

```diff
@@ -217,3 +234,14 @@ func (c *Client) QueryMessageFlags(ctx context.Context, q *QueryOption) (*QueryM
 	err = c.makeRequest(ctx, http.MethodGet, "moderation/flags/message", values, nil, &resp)
 	return &resp, err
 }
+
+type QueryFlagReportsResponse struct {
+	Response
+	FlagReports []*FlagReport `json:"flag_reports"`
+}
+
+func (c *Client) QueryFlagReportsAsync(ctx context.Context, q *QueryFlagReportsRequest) (*QueryFlagReportsResponse, error) {
```

Comment:
why `Async` ?

Reply:
by mistake, sorry. disadvantage of working on 6 languages :)

Reply:
haha no worries, all credits go to you 😊